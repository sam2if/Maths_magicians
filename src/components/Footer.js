import '../App.css';
import './Footer.css';

export default function Footer() {
  return (
    <div className="footer">
      <p>Copyrigth &copy Amazinggracee 2023</p>
    </div>
  );
}
