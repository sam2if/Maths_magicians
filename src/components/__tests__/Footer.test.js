import { render, screen } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer component', () => {
  test('renders the correct copyright message', () => {
    render(<Footer />);
    const copyright = screen.getByText(/Copyrigth &copy Amazinggracee 2023/i);
    expect(copyright).toBeInTheDocument();
  });
});
