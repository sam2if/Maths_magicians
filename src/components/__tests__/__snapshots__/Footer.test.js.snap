import { render, screen } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer component', () => {
  test('renders the correct copyright message', () => {
    render(<Footer />);
    const copyright = screen.getByText(/Copyrigth Â© Amazinggracee 2023/i);
    expect(copyright).toMatchSnapshot();
  });
});
